src/anki/ankiclient.cpp

```diff
@@ -1384,6 +1386,19 @@ void AnkiClient::buildCommonNote(
         value.replace(REPLACE_CLOZE_PREFIX, clozePrefix);
         value.replace(REPLACE_CLOZE_SUFFIX, clozeSuffix);
         value.replace(REPLACE_FREQUENCIES,  frequencies);
+
+        value.replace(REPLACE_FREQ_HARMONIC_RANK, 
+                positiveIntToQString(frequencyHarmonic, 9999999));
+
+        value.replace(REPLACE_FREQ_HARMONIC_OCCU, 
+                positiveIntToQString(frequencyHarmonic, 0));
+
+        value.replace(REPLACE_FREQ_AVERAGE_RANK,  
+                positiveIntToQString(frequencyAverage,  9999999));
```

Comment:
Same as above

Reply:
Fixed in https://github.com/ripose-jp/Memento/pull/238/commits/a92c0c0854dbf417536134dedbb79e2bc0812077